package.json

```diff
@@ -125,7 +128,8 @@
   "bin": {
     "git-info": "./git.info.js",
     "atomist-cli": "./start.cli.js",
-    "atomist-client": "./start.client.js"
+    "atomist-client": "./start.client.js",
+    "atomist-config": "./atomist-config.js"
```

Comment:
I guess you had to pick that name with - so that we wouldn't pick it up as configuration, right?

Reply:
I picked it because it made sense to me have the name of the installed script and its source be the same. Is it customary to have script sources named `start.blah.js`?